mlflow/utils/__init__.py

```diff
@@ -9,6 +9,9 @@
 from contextlib import closing
 from itertools import islice
 from sys import version_info
+from typing import TypeVar
+
+ParamValue = TypeVar("ParamValue")
```

Comment:
This code is not clear, can we delete it from here? If we need it we can use it in the module.

Reply:
Talked on the slack, making this wrapper inline in each file.